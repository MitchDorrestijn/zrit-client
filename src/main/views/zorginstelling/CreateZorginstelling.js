/**
 * React related imports
 */
import React from 'react';

/**
 * Self made components imports
 */
import TextContent from '../../components/TextContent';
import ZorginstellingForm from '../../components/ZorginstellingForm';

/**
 * This component puts the textblock and zorginstellingform (for create) together and renders it
 */
const CreateZorginstelling = () => {
  return (
		<div>
	    <TextContent title="Zorginstelling toevoegen">
	      Op deze pagina kunt u een zorginstelling toevoegen.
				Vul alle velden hieronder in en de nieuwe zorginstelling wordt aan het systeem toegevoegd.
	    </TextContent>
	    <ZorginstellingForm/>
  	</div>
	);
}

export default CreateZorginstelling;
