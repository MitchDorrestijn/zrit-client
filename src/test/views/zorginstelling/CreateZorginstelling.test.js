import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import CreateZorginstelling from '../../../main/views/zorginstelling/CreateZorginstelling';

const wrapper = shallow(<CreateZorginstelling />);

test('<CreateZorginstelling /> should match snapshot', () => {
  expect(toJSON(wrapper)).toMatchSnapshot();
});

test('<CreateZorginstelling /> should render one <TextContent> component', () => {
  expect(wrapper.find('TextContent').length).toBe(1);
});

test('<CreatePage /> should render one <ZorginstellingForm> component', () => {
  expect(wrapper.find('ZorginstellingForm').length).toBe(1);
});
