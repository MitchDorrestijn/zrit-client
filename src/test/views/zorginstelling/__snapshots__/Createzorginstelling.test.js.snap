// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateZorginstelling /> should match snapshot 1`] = `
<div>
  <TextContent
    title="Zorginstelling toevoegen"
  >
    Op deze pagina kunt u een zorginstelling toevoegen. Vul alle velden hieronder in en de nieuwe zorginstelling wordt aan het systeem toegevoegd.
  </TextContent>
  <ZorginstellingForm />
</div>
`;
